### ISSUE TITLE
Installation error on WSL1

### ISSUE THREAD

**Issue by jmontalvo94** [rte-france/Grid2Op#122]
## Environment
 - Grid2op version: `1.1.0`
 - System: `WSL1 (Ubuntu 20.04) on Windows 10`
 - Miniconda version 4.8.3

## Bug description
Installation error appears when trying to install grid2op on WSL1 through PyPI or from source. Solved by commenting out the numba try-except procedure on setup.py, even though numba does get installed.

## How to reproduce
If on a clean conda environment on WSL1, just by trying to install grid2op from PyPI or building it from source.

### Command line
```bash
pip3 install grid2op
pip3 install grid2op --log LOG_FILE.log
```

## Current output
Error appears to be on setup.py of grid2op, since all the other packages are correctly installed on the environment.
```
(l2rpn) jorge@bombe:~$ pip3 install grid2op
Collecting grid2op
  Using cached Grid2Op-1.1.0.tar.gz (11.1 MB)
Requirement already satisfied: numpy>=1.18.3 in ./miniconda3/envs/l2rpn/lib/python3.8/site-packages (from grid2op) (1.19.0)
Collecting scipy>=1.4.1
  Using cached scipy-1.5.1-cp38-cp38-manylinux1_x86_64.whl (25.8 MB)
Collecting pandas>=1.0.3
  Using cached pandas-1.0.5-cp38-cp38-manylinux1_x86_64.whl (10.0 MB)
Processing ./.cache/pip/wheels/f1/5b/06/d7bf05e66b4e0c38e84d44baacc25cc2a85125b9e926fa3a90/pandapower-2.2.2-py3-none-any.whl
Collecting tqdm>=4.45.0
  Using cached tqdm-4.47.0-py2.py3-none-any.whl (66 kB)
Processing ./.cache/pip/wheels/59/02/2f/ff4a3e16a518feb111ae1405908094483ef56fec0dfa39e571/pathlib-1.0.1-py3-none-any.whl
Collecting networkx>=2.4
  Using cached networkx-2.4-py3-none-any.whl (1.6 MB)
Collecting requests>=2.23.0
  Using cached requests-2.24.0-py2.py3-none-any.whl (61 kB)
Collecting python-dateutil>=2.6.1
  Using cached python_dateutil-2.8.1-py2.py3-none-any.whl (227 kB)
Collecting pytz>=2017.2
  Using cached pytz-2020.1-py2.py3-none-any.whl (510 kB)
Collecting packaging
  Using cached packaging-20.4-py2.py3-none-any.whl (37 kB)
Collecting decorator>=4.3.0
  Using cached decorator-4.4.2-py2.py3-none-any.whl (9.2 kB)
Collecting urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1
  Using cached urllib3-1.25.9-py2.py3-none-any.whl (126 kB)
Collecting chardet<4,>=3.0.2
  Using cached chardet-3.0.4-py2.py3-none-any.whl (133 kB)
Requirement already satisfied: certifi>=2017.4.17 in ./miniconda3/envs/l2rpn/lib/python3.8/site-packages (from requests>=2.23.0->grid2op) (2020.6.20)
Collecting idna<3,>=2.5
  Using cached idna-2.10-py2.py3-none-any.whl (58 kB)
Collecting six>=1.5
  Using cached six-1.15.0-py2.py3-none-any.whl (10 kB)
Collecting pyparsing>=2.0.2
  Using cached pyparsing-2.4.7-py2.py3-none-any.whl (67 kB)
Building wheels for collected packages: grid2op
  Building wheel for grid2op (setup.py) ... error
  ERROR: Command errored out with exit status -11:
   command: /home/jorge/miniconda3/envs/l2rpn/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-ml37n_pq/grid2op/setup.py'"'"'; __file__='"'"'/tmp/pip-install-ml37n_pq/grid2op/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-dn13zvn5
       cwd: /tmp/pip-install-ml37n_pq/grid2op/
  Complete output (0 lines):
  ----------------------------------------
  ERROR: Failed building wheel for grid2op
  Running setup.py clean for grid2op
  ERROR: Command errored out with exit status -11:
   command: /home/jorge/miniconda3/envs/l2rpn/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-ml37n_pq/grid2op/setup.py'"'"'; __file__='"'"'/tmp/pip-install-ml37n_pq/grid2op/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' clean --all
       cwd: /tmp/pip-install-ml37n_pq/grid2op
  Complete output (0 lines):
  ----------------------------------------
  ERROR: Failed cleaning build dir for grid2op
Failed to build grid2op
Installing collected packages: scipy, six, python-dateutil, pytz, pandas, pyparsing, packaging, decorator, networkx, pandapower, tqdm, pathlib, urllib3, chardet, idna, requests, grid2op
    Running setup.py install for grid2op ... error
    ERROR: Command errored out with exit status -11:
     command: /home/jorge/miniconda3/envs/l2rpn/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-ml37n_pq/grid2op/setup.py'"'"'; __file__='"'"'/tmp/pip-install-ml37n_pq/grid2op/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-ntaqsgmo/install-record.txt --single-version-externally-managed --compile --install-headers /home/jorge/miniconda3/envs/l2rpn/include/python3.8/grid2op
         cwd: /tmp/pip-install-ml37n_pq/grid2op/
    Complete output (0 lines):
    ----------------------------------------
ERROR: Command errored out with exit status -11: /home/jorge/miniconda3/envs/l2rpn/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-ml37n_pq/grid2op/setup.py'"'"'; __file__='"'"'/tmp/pip-install-ml37n_pq/grid2op/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-ntaqsgmo/install-record.txt --single-version-externally-managed --compile --install-headers /home/jorge/miniconda3/envs/l2rpn/include/python3.8/grid2op Check the logs for full command output.
```

## Log file
```
2020-07-06T15:49:15,022 Non-user install because site-packages writeable
2020-07-06T15:49:15,372 Created temporary directory: /tmp/pip-ephem-wheel-cache-fygfgkwi
2020-07-06T15:49:15,374 Created temporary directory: /tmp/pip-req-tracker-0j64qp3q
2020-07-06T15:49:15,374 Initialized build tracking at /tmp/pip-req-tracker-0j64qp3q
2020-07-06T15:49:15,375 Created build tracker: /tmp/pip-req-tracker-0j64qp3q
2020-07-06T15:49:15,375 Entered build tracker: /tmp/pip-req-tracker-0j64qp3q
2020-07-06T15:49:15,376 Created temporary directory: /tmp/pip-install-t9vrr2ky
2020-07-06T15:49:15,405 1 location(s) to search for versions of grid2op:
2020-07-06T15:49:15,405 * https://pypi.org/simple/grid2op/
2020-07-06T15:49:15,406 Fetching project page and analyzing links: https://pypi.org/simple/grid2op/
2020-07-06T15:49:15,406 Getting page https://pypi.org/simple/grid2op/
2020-07-06T15:49:15,407 Found index url https://pypi.org/simple
2020-07-06T15:50:15,594   Found link https://files.pythonhosted.org/packages/00/47/85286d887e4dfaa310ea99c92a608abb168bd4646747e4c526d0e7c22f90/Grid2Op-0.2.tar.gz#sha256=70ba7aa51efcadbb093de521bfff04383e08400e382b011ec86f63b4b123e2bc (from https://pypi.org/simple/grid2op/), version: 0.2
2020-07-06T15:50:15,594   Found link https://files.pythonhosted.org/packages/74/af/3b219695079cc46f25a1efb48977151c720b86f022313e69efd2ecb951d7/Grid2Op-0.2.1.tar.gz#sha256=07d30f83bd300dc49bd7abf8ae284104f283ed58b38f451e7ee05eea7074e234 (from https://pypi.org/simple/grid2op/), version: 0.2.1
2020-07-06T15:50:15,595   Found link https://files.pythonhosted.org/packages/c5/03/4d3e8f8930e572fbc2c6e9c8c27c383256c3d0981423195879ca159d5293/Grid2Op-0.3.0.tar.gz#sha256=e0c4e7f899a0efeff0b0cb296a3ab2a3554939d112c4ab836fb371f28a33303d (from https://pypi.org/simple/grid2op/), version: 0.3.0
2020-07-06T15:50:15,595   Found link https://files.pythonhosted.org/packages/b8/32/d163d5f837a8a1fd20104e83e7f2f1b696358bb98b40c2679210199e6d91/Grid2Op-0.3.3.tar.gz#sha256=ea918e5b449f4832217a3eb4ae7809814e5d902db6aad8fb2b0fd7a07d66f944 (from https://pypi.org/simple/grid2op/), version: 0.3.3
2020-07-06T15:50:15,595   Found link https://files.pythonhosted.org/packages/d6/04/80e7b8fecce9a46f7f0e8cef7f1421428c7243b02db8c5ab27c0b44c4fd1/Grid2Op-0.3.4.tar.gz#sha256=ba1914ee37566abe556df6eaf5447482fe00b1769fce482bdbc08427e7a311f9 (from https://pypi.org/simple/grid2op/), version: 0.3.4
2020-07-06T15:50:15,595   Found link https://files.pythonhosted.org/packages/d4/5c/bc2751c5f8c5c7ad237031b2fc3fb56796bc08b5ad9f2ad51e4f23151b42/Grid2Op-0.3.5.tar.gz#sha256=70ef02158784c54382365811797d915a2fd2c0c3f4052d21495200707f50d377 (from https://pypi.org/simple/grid2op/), version: 0.3.5
2020-07-06T15:50:15,595   Found link https://files.pythonhosted.org/packages/8b/5b/46863dfb867ceec9e6fc7669233ecfe330ee9a2a8e1c4a12c9f0449aad41/Grid2Op-0.3.6.tar.gz#sha256=75e4df24f4edc9e1059e2458897b891cbf9a79ae682b9de3f994756e813956a3 (from https://pypi.org/simple/grid2op/), version: 0.3.6
2020-07-06T15:50:15,595   Found link https://files.pythonhosted.org/packages/37/a7/1f4253e3037b6813653ae1d41f496fe3eab363981f9576de9754e80325cf/Grid2Op-0.4.0.tar.gz#sha256=b89c1d241770449029e22eb3fa5c66abf37d96413b9cd3030a4a72766b769050 (from https://pypi.org/simple/grid2op/), version: 0.4.0
2020-07-06T15:50:15,596   Found link https://files.pythonhosted.org/packages/09/10/13fc324d19359066d1a2c2f881e163d87c17937d5c9b28dcb0cc1ff9e66f/Grid2Op-0.4.1.tar.gz#sha256=bbd0331ddd4acb695d3a610368b72ee396e86ad399dd54ee59d530b8a261bb4d (from https://pypi.org/simple/grid2op/), version: 0.4.1
2020-07-06T15:50:15,596   Found link https://files.pythonhosted.org/packages/f7/e5/6be87ff8e0c5acce0fb0cb7624a13c6572ebcd5e241a1f886451c3ed463d/Grid2Op-0.4.2.tar.gz#sha256=58f72d1e20c5b39a8130eafa12633df25475c44a405c44a83628e4d4376b5288 (from https://pypi.org/simple/grid2op/), version: 0.4.2
2020-07-06T15:50:15,596   Found link https://files.pythonhosted.org/packages/ad/b0/c193440b0f6ddf1a51ef8c36deb950bfc56073072cb2898503474bc1b997/Grid2Op-0.4.3.tar.gz#sha256=9386a9370a828176e2619e1918b8eb1b6321e578b1e849cdd0abd1d331c2bbc9 (from https://pypi.org/simple/grid2op/), version: 0.4.3
2020-07-06T15:50:15,596   Found link https://files.pythonhosted.org/packages/dd/4c/58204080cf19566006b88d2c4f68220a0a8b8d5479d3cb0e465052b3783a/Grid2Op-0.5.0.tar.gz#sha256=c6f1388fff6d5c6493955d89c40268a99f1882ee7af57f088249aae02ab476e6 (from https://pypi.org/simple/grid2op/), version: 0.5.0
2020-07-06T15:50:15,596   Found link https://files.pythonhosted.org/packages/a6/37/f2374da8719c344f8ebc6798d5f82599bf2f5ad1a7366afbf4e207ade986/Grid2Op-0.5.2.tar.gz#sha256=59930edd4aff07148574109a9641bdb5379eb305e52a246475086baf583e77ef (from https://pypi.org/simple/grid2op/), version: 0.5.2
2020-07-06T15:50:15,597   Found link https://files.pythonhosted.org/packages/65/2a/f9c1b98cac5f8fe754e74ddd26d047a95b664b126eca2d2cece24acc17bb/Grid2Op-0.5.3.tar.gz#sha256=f0afac0d282b042179d73240d0adddbe42ef91b1504800a351f2aac68a731369 (from https://pypi.org/simple/grid2op/), version: 0.5.3
2020-07-06T15:50:15,597   Found link https://files.pythonhosted.org/packages/03/e1/2ec1599f7542fff6f3acdb771f69541525a8bfe461bdea62e690aa1a128b/Grid2Op-0.5.4.tar.gz#sha256=dcf088e4a4fe89a84d8d506d0415d584e2f9ce197d54de200153178b041a4ad7 (from https://pypi.org/simple/grid2op/), version: 0.5.4
2020-07-06T15:50:15,597   Found link https://files.pythonhosted.org/packages/46/3e/044f4d7aa1e500ea26101824ce804fb6b7012dce660c354a6f96b4fdd1ce/Grid2Op-0.5.5.tar.gz#sha256=3aa62c30e2f1f31c17f9771ac2dbf724201d7f74c20388a91a862c7c48b02aef (from https://pypi.org/simple/grid2op/), version: 0.5.5
2020-07-06T15:50:15,597   Found link https://files.pythonhosted.org/packages/f6/8f/f75e0dab80bd351423e0598b48ce02a7fccc0cd339c2520366b262c1e272/Grid2Op-0.5.6.tar.gz#sha256=eaedd40390cfe5335966ed24b2d7c764c5eb19bc20ac5f187ef8de22e4d6a711 (from https://pypi.org/simple/grid2op/), version: 0.5.6
2020-07-06T15:50:15,597   Found link https://files.pythonhosted.org/packages/68/f7/579c4d903aa3d9f5b5b52622b598c9b8b4b7503f65ac12be00688f6dd402/Grid2Op-0.5.7.tar.gz#sha256=d89ff496cd7299bac431a5d3aa107fb231b32f7f98c775cb31127ab290311b46 (from https://pypi.org/simple/grid2op/), version: 0.5.7
2020-07-06T15:50:15,597   Found link https://files.pythonhosted.org/packages/85/05/00a1977b1d87b4afd3d829ccf546d9520dffbcb4cfd61565c0cf62e30bb1/Grid2Op-0.5.8.tar.gz#sha256=99f3bcc433c2169593aab0e5d57f03bc39114c97ae1207d4ee210c3c1064ed03 (from https://pypi.org/simple/grid2op/), version: 0.5.8
2020-07-06T15:50:15,598   Found link https://files.pythonhosted.org/packages/8e/eb/2ca9268dcdc120b31ddc3efd542c75e58f59bc1e14d33f618fcc71a6b54c/Grid2Op-0.6.0.tar.gz#sha256=23599697a31a6143e4b0dd2d2c447b7c7ca1b2cf2d1c18be7745724df2a9fd0d (from https://pypi.org/simple/grid2op/), version: 0.6.0
2020-07-06T15:50:15,598   Found link https://files.pythonhosted.org/packages/f6/bb/b46d530a3b6f3a0ec4a921b2a1674c60cbc45d887bb3c72c70fec1c80918/Grid2Op-0.6.1.tar.gz#sha256=a8c2d6c530d223abed62dd7109efcc86cf56971da87406cd4aeb69d53080697b (from https://pypi.org/simple/grid2op/), version: 0.6.1
2020-07-06T15:50:15,598   Found link https://files.pythonhosted.org/packages/10/01/7b0f21f7783151e3cc2fd27241fe891d1fa6e8c506204d020e0bc0ba2920/Grid2Op-0.7.0.tar.gz#sha256=0861b532d4f30fdb1786b15d4374ced23991b6ccae60938a3d5f400acc14f07b (from https://pypi.org/simple/grid2op/), version: 0.7.0
2020-07-06T15:50:15,598   Found link https://files.pythonhosted.org/packages/c3/6e/29aac36f0a97e472d2a4e3f72320369de2fd52c05ee355066716fa11bafe/Grid2Op-0.7.1.tar.gz#sha256=12466e75ed63f2f3755e6927391e4fd95fabee031e243a086c53065731645f03 (from https://pypi.org/simple/grid2op/), version: 0.7.1
2020-07-06T15:50:15,598   Found link https://files.pythonhosted.org/packages/0a/85/4298650b3bcd9a63c141baf7affe5f99583dc3150113adddccbb9898cbe5/Grid2Op-0.8.0.tar.gz#sha256=106ee47c551988868b16ac2d1582e978980474c56644401d3ab7912319163d54 (from https://pypi.org/simple/grid2op/), version: 0.8.0
2020-07-06T15:50:15,598   Found link https://files.pythonhosted.org/packages/4c/53/c6371b7a24a6697b248f599cdc44b81386335ed30e80b82272f6991af2c3/Grid2Op-0.8.1.tar.gz#sha256=573859587948ea075e996e6ea207db84c76686a3bd2e044622eebd5f8c19b6fd (from https://pypi.org/simple/grid2op/), version: 0.8.1
2020-07-06T15:50:15,599   Found link https://files.pythonhosted.org/packages/2d/ef/656dec04cadf96a075d155fdc379704df131e9850ddee039b16ecfbbe36f/Grid2Op-0.8.2.tar.gz#sha256=10b005f38f0c72a42422f5bdfe90cfd34563ef5a5239bd7349ab925ad5bb9a77 (from https://pypi.org/simple/grid2op/), version: 0.8.2
2020-07-06T15:50:15,599   Found link https://files.pythonhosted.org/packages/32/4e/12e29bf70acdbe38ddf34817ee080a5b4effab1a79f5d7cb9f5a7c74a3ea/Grid2Op-0.9.0.tar.gz#sha256=5b05dbe73650f41990d8fd28e14d689695b09d3415a0cbde3906892bf40c06f7 (from https://pypi.org/simple/grid2op/), version: 0.9.0
2020-07-06T15:50:15,599   Found link https://files.pythonhosted.org/packages/31/d7/091d87616d06501ec9ac3ee4a404e0e86624924618ed52cb45c39e3f899c/Grid2Op-0.9.1.tar.gz#sha256=116024776eaf4fdf7bb4367e65b77ead2b3c897922916d7e608b723d51a00fab (from https://pypi.org/simple/grid2op/), version: 0.9.1
2020-07-06T15:50:15,599   Found link https://files.pythonhosted.org/packages/86/52/19eb6bb3cb02078834a0468181887c86ab87e7468dd3bff22b8e101484d5/Grid2Op-0.9.1.post1.tar.gz#sha256=75c62548a19af415f4b661dcfa5cac7706936aee9f4a8b7c05422ad86edbf237 (from https://pypi.org/simple/grid2op/), version: 0.9.1.post1
2020-07-06T15:50:15,599   Found link https://files.pythonhosted.org/packages/26/d3/5c9c52c0928d45a6a72ec5fc994670ede785a87f5f932dc1c79f156f4214/Grid2Op-0.9.2.tar.gz#sha256=a590394c93b423a0b71f9de933e72b0461a309819746ae2d84001c1c3dc2713d (from https://pypi.org/simple/grid2op/), version: 0.9.2
2020-07-06T15:50:15,599   Found link https://files.pythonhosted.org/packages/0a/50/9511bafee6c5c77c38b70c50b3ed0d0b18a02573192f42fc132dd9fde977/Grid2Op-0.9.3.tar.gz#sha256=8b743830b1ad02f9e815eb2ad0dee5c5bbf5e5fb9290cfc513c65f950ee36c40 (from https://pypi.org/simple/grid2op/), version: 0.9.3
2020-07-06T15:50:15,600   Found link https://files.pythonhosted.org/packages/71/d4/ab7dcec2945db8d319b2a2ec7455ec5e076ab4baa524abab89cc34568624/Grid2Op-0.9.4.tar.gz#sha256=73ef92935db47c136125b253dc33ba4798a2e7b384beec551bfcafae7ed6a580 (from https://pypi.org/simple/grid2op/), version: 0.9.4
2020-07-06T15:50:15,600   Found link https://files.pythonhosted.org/packages/a3/e0/a16d8be7e88cc8d4032aa24adea5dd6cd4e7564587c635292f8cbc645d4e/Grid2Op-1.0.0.tar.gz#sha256=6d88d8cf6eacbfccb8eb758d2c70e113f44bccc37df3701fdccf9de9ea762de1 (from https://pypi.org/simple/grid2op/), version: 1.0.0
2020-07-06T15:50:15,600   Found link https://files.pythonhosted.org/packages/d4/9b/60cd2985b2d2747d527e6e1c1e6951d9753781c31152c92652d2b5a4e7ca/Grid2Op-1.1.0.tar.gz#sha256=c574c07221a0d6a9a5af17d829bb3de63e26d3cb8dee78c7d9ac062e671f72e7 (from https://pypi.org/simple/grid2op/), version: 1.1.0
2020-07-06T15:50:15,602 Given no hashes to check 34 links for project 'grid2op': discarding no candidates
2020-07-06T15:50:15,605 Using version 1.1.0 (newest of versions: 0.2, 0.2.1, 0.3.0, 0.3.3, 0.3.4, 0.3.5, 0.3.6, 0.4.0, 0.4.1, 0.4.2, 0.4.3, 0.5.0, 0.5.2, 0.5.3, 0.5.4, 0.5.5, 0.5.6, 0.5.7, 0.5.8, 0.6.0, 0.6.1, 0.7.0, 0.7.1, 0.8.0, 0.8.1, 0.8.2, 0.9.0, 0.9.1, 0.9.1.post1, 0.9.2, 0.9.3, 0.9.4, 1.0.0, 1.1.0)
2020-07-06T15:50:15,607 Collecting grid2op
2020-07-06T15:50:15,609   Created temporary directory: /tmp/pip-unpack-g4ytc442
2020-07-06T15:50:15,741   Using cached Grid2Op-1.1.0.tar.gz (11.1 MB)
2020-07-06T15:50:17,849   Added grid2op from https://files.pythonhosted.org/packages/d4/9b/60cd2985b2d2747d527e6e1c1e6951d9753781c31152c92652d2b5a4e7ca/Grid2Op-1.1.0.tar.gz#sha256=c574c07221a0d6a9a5af17d829bb3de63e26d3cb8dee78c7d9ac062e671f72e7 to build tracker '/tmp/pip-req-tracker-0j64qp3q'
2020-07-06T15:50:17,849     Running setup.py (path:/tmp/pip-install-t9vrr2ky/grid2op/setup.py) egg_info for package grid2op
2020-07-06T15:50:17,850     Created temporary directory: /tmp/pip-pip-egg-info-h09kncl9
2020-07-06T15:50:17,850     Running command python setup.py egg_info
2020-07-06T15:50:18,909 ERROR: Command errored out with exit status -11: python setup.py egg_info Check the logs for full command output.
2020-07-06T15:50:18,911 Exception information:
2020-07-06T15:50:18,911 Traceback (most recent call last):
2020-07-06T15:50:18,911   File "/home/jorge/miniconda3/envs/l2rpn/lib/python3.8/site-packages/pip/_internal/cli/base_command.py", line 188, in _main
2020-07-06T15:50:18,911     status = self.run(options, args)
2020-07-06T15:50:18,911   File "/home/jorge/miniconda3/envs/l2rpn/lib/python3.8/site-packages/pip/_internal/cli/req_command.py", line 185, in wrapper
2020-07-06T15:50:18,911     return func(self, options, args)
2020-07-06T15:50:18,911   File "/home/jorge/miniconda3/envs/l2rpn/lib/python3.8/site-packages/pip/_internal/commands/install.py", line 332, in run
2020-07-06T15:50:18,911     requirement_set = resolver.resolve(
2020-07-06T15:50:18,911   File "/home/jorge/miniconda3/envs/l2rpn/lib/python3.8/site-packages/pip/_internal/resolution/legacy/resolver.py", line 179, in resolve
2020-07-06T15:50:18,911     discovered_reqs.extend(self._resolve_one(requirement_set, req))
2020-07-06T15:50:18,911   File "/home/jorge/miniconda3/envs/l2rpn/lib/python3.8/site-packages/pip/_internal/resolution/legacy/resolver.py", line 362, in _resolve_one
2020-07-06T15:50:18,911     abstract_dist = self._get_abstract_dist_for(req_to_install)
2020-07-06T15:50:18,911   File "/home/jorge/miniconda3/envs/l2rpn/lib/python3.8/site-packages/pip/_internal/resolution/legacy/resolver.py", line 314, in _get_abstract_dist_for
2020-07-06T15:50:18,911     abstract_dist = self.preparer.prepare_linked_requirement(req)
2020-07-06T15:50:18,911   File "/home/jorge/miniconda3/envs/l2rpn/lib/python3.8/site-packages/pip/_internal/operations/prepare.py", line 487, in prepare_linked_requirement
2020-07-06T15:50:18,911     abstract_dist = _get_prepared_distribution(
2020-07-06T15:50:18,911   File "/home/jorge/miniconda3/envs/l2rpn/lib/python3.8/site-packages/pip/_internal/operations/prepare.py", line 91, in _get_prepared_distribution
2020-07-06T15:50:18,911     abstract_dist.prepare_distribution_metadata(finder, build_isolation)
2020-07-06T15:50:18,911   File "/home/jorge/miniconda3/envs/l2rpn/lib/python3.8/site-packages/pip/_internal/distributions/sdist.py", line 40, in prepare_distribution_metadata
2020-07-06T15:50:18,911     self.req.prepare_metadata()
2020-07-06T15:50:18,911   File "/home/jorge/miniconda3/envs/l2rpn/lib/python3.8/site-packages/pip/_internal/req/req_install.py", line 550, in prepare_metadata
2020-07-06T15:50:18,911     self.metadata_directory = self._generate_metadata()
2020-07-06T15:50:18,911   File "/home/jorge/miniconda3/envs/l2rpn/lib/python3.8/site-packages/pip/_internal/req/req_install.py", line 525, in _generate_metadata
2020-07-06T15:50:18,911     return generate_metadata_legacy(
2020-07-06T15:50:18,911   File "/home/jorge/miniconda3/envs/l2rpn/lib/python3.8/site-packages/pip/_internal/operations/build/metadata_legacy.py", line 70, in generate_metadata
2020-07-06T15:50:18,911     call_subprocess(
2020-07-06T15:50:18,911   File "/home/jorge/miniconda3/envs/l2rpn/lib/python3.8/site-packages/pip/_internal/utils/subprocess.py", line 241, in call_subprocess
2020-07-06T15:50:18,911     raise InstallationError(exc_msg)
2020-07-06T15:50:18,911 pip._internal.exceptions.InstallationError: Command errored out with exit status -11: python setup.py egg_info Check the logs for full command output.
2020-07-06T15:50:19,057 Removed grid2op from https://files.pythonhosted.org/packages/d4/9b/60cd2985b2d2747d527e6e1c1e6951d9753781c31152c92652d2b5a4e7ca/Grid2Op-1.1.0.tar.gz#sha256=c574c07221a0d6a9a5af17d829bb3de63e26d3cb8dee78c7d9ac062e671f72e7 from build tracker '/tmp/pip-req-tracker-0j64qp3q'
2020-07-06T15:50:19,057 Removed build tracker: '/tmp/pip-req-tracker-0j64qp3q'
```

**Comment by BDonnot** [comment]
Thanks :-)

This issue will be fixed in the next release.
And thanks again for your time and availability in discord :-)

Benjamin

**Comment by BDonnot** [comment]
New fix has been made and pushed in version 1.1.1
